src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -55,72 +55,81 @@ Need to edit a doc? Use this table to decide where to work!
-
-1. First, ensure your **Current Branch** in GitHub Desktop is set to the correct branch, not **Develop**.
-2. Navigate to the doc you want to edit in Finder. If I wanted to edit a Python agent doc, I would navigate to: `~/Documents/github/docs-website/src/content/docs/agents/python-agent/hosting-services/python-agent-stackato.mdx`
-3. Edit the doc in your text editor of choice. You should write docs in markdown language. Reference the style guide for help with formatting markdown
-4. Save the file with your edits, then follow the same process for any other docs you wish to edit.
-
-## 4. Commit your changes [#commit-changes]
-Once your edits are done, you can commit them. This stages your changes, which you will later push upstream to Github. By pushing your changes, everyone will have access to your branch and commits. 
-
-1. Navigate to GitHub Desktop. The left column should have a record of all the edits you have made to docs. 
-2. In the bottom left corner, name your commit and add a good description of your edits. It should be descriptive enough to ensure that someone can understand all the changes made by simply scanning this description.
-3. Click **Commit to [yourbranchname]**
+4. Once built, preview the site in your browser by navigating to [http://localhost:8000/](http://localhost:8000/)
 
 ## 5. Publish your commits [#publish-commits]
-
-Once you have committed your changes, you're almost ready to open your Pull Request.
-
-First, you need to ensure your branch is pushed upstream.
+Once you're satisfied with doc, you'll want to push all of your commits upstream. This makes them available to everyone else in the GitHub repository. 
 
 1. On GitHub desktop, click the blue **Publish Branch** button if available.
 2. If you don't see the Publish Branch, click the blue **Push Origin** button. 
 
-This will push all your commits upstream and make them available to everyone else through the GitHub repository.
+Now you're ready for a Pull Request. 
 
 ## 6. Open your pull request [#open-pr]
-Now that your commits are available to everyone, you need to notify people that your changes are ready to be merged into the `develop` branch.
+Your commits are available to everyone, but you'll need to notify people that your changes are ready to be merged into the `develop` branch.
 
-To do this, you open a pull request:
+To do this, you open a pull request (PR):
 
 1. On GitHub Desktop, click the blue **Create Pull Request button**.
-2. This will open GitHub in your browser, and prompt you to fill in your pull request. Ensure you're merging from your branch into either the `main` or `develop` branch. If you scroll down, you can review all your commits to ensure they reflect all your changes.
+2. This will open GitHub in your browser, and prompt you to fill in your pull request. 
+    - Ensure you're merging from your branch into either the `main` or `develop` branch.
+    - Scroll down to review all your commits. They should reflect all your changes.
```

Comment:
```suggestion
    - Scroll down to review all your commits.
```

There's no "should" about it. All of your changes are there. I've never known this process to fail.